rules.mk.j2

```diff
@@ -118,11 +101,9 @@ deploy-to-s3: package
 		sed 's| (.*) | (latest.tar.gz) |' build/osie-$v.tar.gz.sha512sum >build/latest.tar.gz.sha512sum
 		mc cp s3/tinkerbell-oss/osie-uploads/latest.tar.gz.sha512sum s3/tinkerbell-oss/osie-uploads/latest.tar.gz.sha512sum
 	fi
-	echo "deploy this build from the deploy-osie repo with the following command:"
-	echo -n "./deploy update osie-$v -m \"$$"
-	echo -n "(sed -n '/^## \[/,$$ {/\S/!q; p}' CHANGELOG.md)\" "
-	echo -n "$$(sed 's|.*= ||' build/osie-$v.tar.gz.sha512sum)"
-	echo
```

Comment:
I regularly rely on this info when deploying osie, and I know I've taught several other folks to deploy osie using this info as reference. I realize it's EM specific but I'd really like to keep that to avoid further hassles.

Reply:
yeah I was on the fence about it, let me re-word it instead.

Reply:
I've updated the messaging a bit more, PTAL again.